stacks/airflow/airflow.yaml

```diff
@@ -33,7 +33,7 @@ spec:
           min: 400m
           max: "1"
         memory:
-          limit: 1Gi
+          limit: 2Gi
```

Comment:
The is apparently the default, so maybe we should remove it here?

Reply:
I would not since we provide calculations about overall required resources for each demo...